### PR TITLE
Change size of activity diagrams

### PR DESCRIPTION
**Pull request by TaoJun99**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -679,7 +671,7 @@ name with that assignment.
 The following activity diagram summarizes what happens when a user executes the done command:
 
 <p align="center">
-  <img src="images/DoneActivityDiagram.png" width="250" />
+  <img src="images/DoneActivityDiagram.png" width="600" height="550" />
```
Comment (qinguorui2001) [inline docs/DeveloperGuide.md:674]: so some activity diagrams have width 500 some have 600, is it intentional?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -155,14 +155,6 @@ The `Model` component,
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
-
-  <p align="center">
```
Comment (qinguorui2001) [inline docs/DeveloperGuide.md:160]: good delete!

**Review by qinguorui2001 — COMMENTED** [review]
ask for clarification

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (TaoJun99) [reply docs/DeveloperGuide.md:674]: Yes! Some activity diagrams have longer sentences, so a bigger width will make the words easier to read!

**Review by yeo-yiheng — APPROVED** [review]
LGTM!

**Pull request merged**